client/app/templates/json-api-errors.hbs

```diff
@@ -0,0 +1,9 @@
+{{#if errors}}
```

Comment:
JsonApi defines how errors comeback

```json
{
  "errors": [
    "title": "title of error",
    "detail": "..."
  ]
}
```
right now I set a property on the component [here](https://github.com/voluspa/yggdrasil/pull/14/files#diff-8fd9ef2cf4b5a84c508b29537979521cR23), which pulls out the errors, but I think maybe I should have set the model and the template use the model. I think that is what the ember data stuff will be doing as well then this template would 

```
{{#if model.errors}}
```

and it makes this even more reusable I think.